mmocr/datasets/utils/backend.py

```diff
@@ -40,15 +41,33 @@ def __getitem__(self, index):
             self.env = self._get_env()
 
         with self.env.begin(write=False) as txn:
-            # this is for label only lmdb format
-            try:
-                line = txn.get(str(index).encode('utf-8')).decode(
-                    self.encoding)
-            # this is for image and label lmdb format
-            except Exception:
-                index = index + 1
-                label_key, filename = b'label-%09d' % index, r'%s' % index
-                line = filename + ' ' + txn.get(label_key).decode()
+            # use initial_item to avoid error triggered frequently
+            if self.initial_item:
+                # this is for label only lmdb format
+                try:
+                    line = txn.get(str(index).encode('utf-8')).decode(
+                        self.encoding)
+                    self.label_only = True
```

Comment:
Can we move the assignment of this variable to __init__? We already know the format when retrieving `total_number` or `num-samples` from the dataset.

Reply:
We can't tell if label_only from `total_numer` or `num-samples`. In `lmdb_converter`, whether it is label_only or not, I am using `total_number` to be uniform with the previous MMOCR's data.